### ISSUE TITLE
[Bug] Themes not applying to Enmity

### ISSUE THREAD

**Issue by korboybeats** [s8ngyu/Mugunghwa#17]
I have tried Elusive, modernOS, and Darker and they all do not apply to Enmity, but if I install normal Discord, it works.

Device: M1 iPad Pro
iOS: 15.5 Beta 4

Darker icons applied here:

![HbiZW4B](https://user-images.githubusercontent.com/80824905/197340039-d1aca666-2dbd-4b7a-8264-0e8d1ff05c7b.jpg)

**Comment by swayea** [comment]
probably theme searchs for discord, not enmity. Identifiers are different like;

com.xxxx.discord com.xxxx.enmity

**Comment by korboybeats** [comment]
> probably theme searchs for discord, not enmity. Identifiers are different like;
> 
> com.xxxx.discord com.xxxx.enmity

I checked the identifier for Enmity. It is the same as normal Discord, `com.hammerandchisel.discord`

**Comment by swayea** [comment]
yes i see. i changed icon manually and not any problem occured. maybe you need rebuild icon cache from trollstore or if u have newterm just do /uicache -r

**Comment by korboybeats** [comment]
> yes i see. i changed icon manually and not any problem occured. maybe you need rebuild icon cache from trollstore or if u have newterm just do /uicache -r

Yeah I've rebuilt icon cache several times, and unfortunately I'm not jailbroken so I don't have Newterm

**Comment by swayea** [comment]
Well, the app icon name in enmity counts as EnmityAppIcon, not AppIcon. Maybe something different here like, if you upload a '' AppIcon '' named icon, its not get changed. But if you upload EnmityAppIcon it gets changed. idk how 2.9 works but i think i am true.

**Comment by korboybeats** [comment]
> Well, the app icon name in enmity counts as EnmityAppIcon, not AppIcon. Maybe something different here like, if you upload a '' AppIcon '' named icon, its not get changed. But if you upload EnmityAppIcon it gets changed. idk how 2.9 works but i think i am true.

That workeddd. Changed all "EnmityIcon" to "AppIcon" and now it works. TY

**Comment by swayea** [comment]
> > Well, the app icon name in enmity counts as EnmityAppIcon, not AppIcon. Maybe something different here like, if you upload a '' AppIcon '' named icon, its not get changed. But if you upload EnmityAppIcon it gets changed. idk how 2.9 works but i think i am true.
> 
> That workeddd. Changed all "EnmityIcon" to "AppIcon" and now it works. TY

yrw! :)
